table WO_TEST_EQUIP


    column 'SITE'
        dataType: string
        summarizeBy: none
        sourceColumn: SITE

        

        annotation SummarizationSetBy = Automatic

        
    column 'WORKORDER'
        dataType: string
        summarizeBy: none
        sourceColumn: WORKORDER

        

        annotation SummarizationSetBy = Automatic

        
    column 'REC_TYP'
        dataType: string
        summarizeBy: none
        sourceColumn: REC_TYP

        

        annotation SummarizationSetBy = Automatic

        
    column 'EQUIP_NUM'
        dataType: string
        summarizeBy: none
        sourceColumn: EQUIP_NUM

        

        annotation SummarizationSetBy = Automatic

        



    partition 'WO_TEST_EQUIP' = m
        mode: import
        source = 
            // ERROR: Error communicating with LLM service for table WO_TEST_EQUIP: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /api/m-query (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x102f95580>: Failed to establish a new connection: [Errno 61] Connection refused'))
let
				Source = Table.FromRows({})
				in
				Source
        



    annotation PBI_ResultType = Table